### PR TITLE
Update MAO and openshift/api dependency

### PR DESCRIPTION
**Pull request by mkumatag**
Updated the dependencies to latest levels.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,3 +24,104 @@ require (
 	sigs.k8s.io/controller-tools v0.6.3-0.20210916130746-94401651a6c3
 	sigs.k8s.io/yaml v1.2.0
 )
+
+require (
+	cloud.google.com/go v0.81.0 // indirect
```
Comment (Karthik-K-N) [inline go.mod:29]: Why we have 2 require blocks here and so many indirect dependencies. Also if we try `go get github.com/openshift/api@master` we may get even recent api changes

Comment (mkumatag) [reply go.mod:29]: The second block got added part of golang 1.17 update in the go.mod file, have seen this pattern everywhere who is using 1.17 version

can try updating to the latest available version of api and mapio

Comment (mkumatag) [reply go.mod:29]: @Karthik-K-N done updating to the latest available `openshift/api` repo, ptal

**Pull request merged**